pkg/controllers/podgroup/pg_controller_handler.go

```diff
@@ -49,6 +49,24 @@ func (pg *pgcontroller) addPod(obj interface{}) {
 	pg.queue.Add(req)
 }
 
+func (pg *pgcontroller) deletePG(obj interface{}) {
+	pod, ok := obj.(*v1.Pod)
+	if !ok {
+		klog.Errorf("Failed to convert %v to v1.Pod", obj)
+		return
+	}
+
+	pgName := helpers.GeneratePodgroupName(pod)
+	if _, err := pg.pgLister.PodGroups(pod.Namespace).Get(pgName); err != nil {
+		return
+	}
+
+	if err := pg.vcClient.SchedulingV1beta1().PodGroups(pod.Namespace).Delete(context.TODO(), pgName, metav1.DeleteOptions{}); err != nil {
+		klog.Errorf("Failed to delete volcano job,ns:%s,name:%s,err:", pod.Namespace, pgName, err)
+		return
+	}
+}
+
```

Comment:
i'm ok about the changes, but is there a way to filter out the pods under vcjob? Because currently all pods are watched, but when the pods under vcjob are deleted, deleting pg will throw an error

Reply:
Agree with @shinytang6 .Without the filter, just delete the pg is not so reasonable when some pod is deleted.